docs/docs/09.1-language-tooling.md

```diff
@@ -74,6 +74,9 @@ Flow is a JavaScript type checker released by Facebook, and it supports JSX. For
 
 ## TypeScript
 
-Microsoft TypeScript now supports JSX. For more info, check out the [TypeScript JSX Documentation](http://www.typescriptlang.org/docs/handbook/jsx.html) or their guide on [getting started with react+webpack](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/tutorials/React%20%26%20Webpack.md).
+TypeScript is a type-checker and transpiler that supports type-checking React and JSX.
+For more info, check out their guide on [getting started with React and Webpack](www.typescriptlang.org/docs/handbook/react-&-webpack.html), or learn more about [TypeScript's JSX support](http://www.typescriptlang.org/docs/handbook/jsx.html).
```

Comment:
Can you fix the first link so it has a protocol? Also, looks like your docs use https, want to point to that for all of these links?